src/Tribe/Integrations/ACF/ACF.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Class Tribe__Events__Integrations__ACF__ACF
+ *
+ * Handles the integration between The Events Calendar plugin and Advanced Custom Fields.
+ *
+ * This class is meant to be an entry point hooking specialized classes and not
+ * a logic hub per se.
+ */
+class Tribe__Events__Integrations__ACF__ACF {
+
+	/**
+	 * @var Tribe__Events__Integrations__ACF__ACF
+	 */
+	protected static $instance;
```

Comment:
What about using DI52 here instead?

Reply:
i don't think it would add much value and for the sake of time am going to leave as-is — confirmed with barry ("link":https://cloudup.com/cAGITw95CDU)

Reply:
No worries! It just saves time and deprecating to do it at the start, so I figured I'd bring it up in case.